threedigrid_builder/interface/db.py

```diff
@@ -207,17 +203,37 @@ def get_settings(self) -> dict:
         if infiltration:
             _set_initialization_type(infiltration, "infiltration_rate", default=NO_AGG)
             _set_initialization_type(
-                infiltration, "max_infiltration_capacity", default=NO_AGG
+                infiltration, "max_infiltration_volume", default=NO_AGG
             )
-
         if groundwater:
             # default is what the user supplied (MIN/MAX/AVERAGE)
-            _set_initialization_type(groundwater, "groundwater_impervious_layer_level")
-            _set_initialization_type(groundwater, "phreatic_storage_capacity")
+
+            _set_initialization_type(
```

Comment:
This method `_set_initialization_type`, I think you now always supply `type_field` to it? Maybe good to remove the string logic over there accordingly.

Reply:
No, there in most cases the `type_field` is not specified. I'll keep an eye on this in further migrations and remove the logic when it's no longer (or almost no longer) in use.